apis/acmpca/acmpca.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package certificatemanager contains certificatemanager API versions
+package certificatemanager
```

Comment:
```suggestion
// Package acmpa contains private certificate manager API versions
package acmpca
```

Reply:
@hasheddan  Rebased with upstream master and updated build submodule too.
Resolved the type review comments.
Kindly review the code and let me know if any further changes required.